authorization/src/main/java/com/terabite/authorization/AuthorizationController.java

```diff
@@ -0,0 +1,39 @@
+package com.terabite.authorization;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class AuthorizationController {
+
+    @GetMapping("/user/signup")
+    public Payload userSignup() {
+        return new Payload("Reached signup");
+    }
+
+    @PostMapping("/user/signup")
+    public Payload userSignupPost() {
+        return new Payload("Reached signup POST");
+    }
+
+    @GetMapping("/user/login")
+    public Payload userLogin() {
+        return new Payload("Reached login");
+    }
+
+    @PostMapping("/user/login")
```

Comment:
I think we only need a POST endpoint for login, no need to have a get endpoint. Can we make sure that we are versioning this endpoint as well. So `v1/user/login` for example. 

And lets Rename the payload to something ending with Request (for request payloads) and Response for response payloads. 

So `LoginResponse.java` would be the payload we respond with to login requests.

Reply:
Will add versioning to routes. Are we gonna have some kind of "endpoint regulation" docs where endpoints have to follow rules like starting w/ `v1/`?

Ans for payload's the same as prev comment

Reply:
I don't know if we have anything written up, but it is in one of the videos. I'll try to get that posted to the repo documentation sometime this weekend.